plugins/plugin-chart-pivot-table/src/plugin/transformProps.ts

```diff
@@ -108,7 +107,7 @@ export default function transformProps(chartProps: ChartProps<QueryFormData>) {
     .reduce((acc: Record<string, DateFormatter | undefined>, temporalColname: string) => {
       let formatter: DateFormatter | undefined;
       if (dateFormat === smartDateFormatter.id) {
-        if (temporalColname === TIME_COLUMN) {
+        if (granularity) {
```

Comment:
Should we also ensure that the column type is temporal? Otherwise we will end up applying the temporal formatter on all groupbys if a time grain and smart date formatter is selected.

Reply:
We do that in line 106 :) 
`.filter((colname: string, index: number) => coltypes[index] === GenericDataType.TEMPORAL)`

Reply:
Lol nvm!